core/src/main/java/org/apache/accumulo/core/file/rfile/bcfile/BoundedRangeFileInputStream.java

```diff
@@ -95,18 +92,7 @@ public int read(final byte[] b, final int off, int len) throws IOException {
         throw new IOException("Stream closed");
       }
       in.seek(pos);
-      try {
-        ret = AccessController.doPrivileged(new PrivilegedExceptionAction<Integer>() {
```

Comment:
Ahh, ok. Great. Not being able to attend that hackathon kept me in the dark, I guess :)